modules/ztp-sno-du-configuring-disk-encryption-with-pcr-protection.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-sno-du-configuring-disk-encryption-with-pcr-protection_{context}"]
+= Enabling disk encryption with TPM and PCR protection
+
+You can use the `diskEncryption` field in the `SiteConfig` custom resource (CR) to configure disk encryption with Trusted Platform Module (TPM) and Platform Configuration Registers (PCRs) protection. 
+
+Configuring the `SiteConfig` CR enables disk encryption at cluster installation time.
+
+.Prerequisites
+
+* You have installed the {oc-first}.
+* You have logged in as a user with `cluster-admin` privileges.
+* You read the "About disk encryption with PCR protection" section.
+
+.Procedure
+
+* Configure the `spec.clusters.diskEncryption` field in the `SiteConfig` CR:
++
+.Recommended `SiteConfig` CR configuration to enable disk encryption with PCR protection
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "encryption-tpm2"
+  namespace: "encryption-tpm2"
+spec:
+  clusters:
+  - clusterName: "encryption-tpm2"
+    clusterImageSetNameRef: "openshift-v4.13.0"
+    diskEncryption: 
+      type: "tpm2" <1>
+      tpm2:
+        pcrList: "1,7" <2>
+    nodes:
+      - hostName: "node1"
+        role: master
+----
+<1> Set the disk encryption type to `tpm2`.
+<2> Configure the list of PCRs to be used for disk encryption. You must use PCR registers 1 and 7.
```

Comment:
```suggestion
<2> Configures the list of PCRs to be used for disk encryption. You must use PCR registers 1 and 7.
```

Reply:
Otherwise these sentences sound contradictory to me.

Reply:
I think "Configure" would sound appropriate in this context as we are asking the user to do it. "Configures" sounds more like an action that is completed automatically.
The second sentence talks about which registers must be used to configure the list.